src/pyfaf/repos/Makefile.am

```diff
@@ -1,6 +1,17 @@
 repos_PYTHON = \
     __init__.py \
-    yum.py \
     rpm_metadata.py
 
+if USE_DNF
+    repos_PYTHON += dnf.py
+endif
+
+if USE_YUM
+    repos_PYTHON += yum.py
+endif
+
+EXTRA_DIST = $(repos_PYTHON)
+
+am__repos_PYTHON_DIST = $(repos_PYTHON)
```

Comment:
Then there are built both dnf.py and yum.py files which is unwanted.

Reply:
Not reproduced, by:
```
$ ( rm -rf i && ./autogen.sh && ./configure --without-yum --with-dnf && make install DESTDIR=`pwd`/i ) &>/dev/null
$ find i | grep -e dnf -e yum
i/usr/local/etc/faf/plugins/dnf.conf
i/usr/local/lib/python2.7/site-packages/pyfaf/repos/dnf.py
i/usr/local/lib/python2.7/site-packages/pyfaf/repos/dnf.pyc
i/usr/local/lib/python2.7/site-packages/pyfaf/repos/dnf.pyo
i/usr/local/lib/python2.7/site-packages/pyfaf/storage/migrations/versions/e5d5cefb8ca4_add_dnf_type.py
i/usr/local/lib/python2.7/site-packages/pyfaf/storage/migrations/versions/e5d5cefb8ca4_add_dnf_type.pyc
i/usr/local/lib/python2.7/site-packages/pyfaf/storage/migrations/versions/e5d5cefb8ca4_add_dnf_type.pyo
```

Reply:
@ari3s can you confirm that `yum.pyc` is compiled at `make install` time on your box?  It would be worth filling a bug report against proper distribution automake package

(sorry, I didn't say this -> this patch was applied before running ./autogen.sh)
```patch
diff --git a/src/pyfaf/repos/Makefile.am b/src/pyfaf/repos/Makefile.am
index a4aeaaa4..c9352bf3 100644
--- a/src/pyfaf/repos/Makefile.am
+++ b/src/pyfaf/repos/Makefile.am
@@ -12,6 +12,4 @@ endif
 
 EXTRA_DIST = $(repos_PYTHON)
 
-am__repos_PYTHON_DIST = $(repos_PYTHON)
-
 reposdir = $(pythondir)/pyfaf/repos
```